bot/exts/moderation/infraction/management.py

```diff
@@ -262,6 +262,44 @@ async def search_reason(self, ctx: Context, reason: str) -> None:
         )
         await self.send_infraction_list(ctx, embed, infraction_list)
 
+    # endregion
+    # region: Search infractions by given user
+    @infraction_group.command(name="by", aliases=("b",))
+    async def search_by_user(
```

Comment:
I know what you meant here, but this reads like the function is making a search by user, and not searching for something made by a user. I think `search_by_actor` is a better name here, and then change the argument name to `actor` accordingly.

Reply:
Will change :+1: